README.md

```diff
@@ -1,2 +1,24 @@
-# trinity-carpool
-A carpool management app by the students of Austin's Trinity Episcopal School.
+# Trinity Carpool
+### A carpool management app designed and created by the students of Austin's Trinity Episcopal School. 
+-------------------------------------------------------------------------------
```

Comment:
why is this line longer? lengthen the others or make this shorter

Reply:
I agree, lets do this!

Reply:
I agree! We should make this shorter. Good idea!